lib/pages/opened_recipe_page.dart

```diff
@@ -0,0 +1,323 @@
+            child: ListTile(
+              title: Column(
+                mainAxisAlignment: MainAxisAlignment.start,
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: <Widget>[
+                  Text(getProductName(recipeItems[index].label)),
+                  Text(getProductBrand(recipeItems[index].label)),
+                ],
+              ),
+              subtitle: Row(
+                children: <Widget>[
+                  Text(
+                      "x${recipeItems[index].quantity.toString()} ${getUnitText(recipeItems[index].unit)}"),
+                ],
+              ),
+              leading: Image(image: getItemImage(recipeItems[index].imageUrl)),
+              trailing: Icon(
+                Icons.navigate_next,
+                color: BLACK,
+              ),
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+
+  buildDeleteShowDialog(BuildContext context) {
+    return showDialog(
+      context: context,
+      builder: (BuildContext context) {
+        return AlertDialog(
+          content: Text(getString(context, 'confirm_item_deletion')),
+          actions: <Widget>[
+            FlatButton(
+                onPressed: () => Navigator.of(context).pop(true),
+                child: Text(getString(context, 'delete_item'))),
+            FlatButton(
+              onPressed: () => Navigator.of(context).pop(false),
+              child: Text(getString(context, 'cancel_deletion')),
+            ),
+          ],
+        );
+      },
+    );
+  }
+
+  void deleteItemFromRecipe(
+      {String recipeUuid, String itemUuid, String imageName}) {
+    GetIt.I.get<RecipeItemsProvider>().deleteItemInRecipe(
+        recipeUuid: recipeUuid, itemUuid: itemUuid, imageName: imageName);
+  }
+
+  dynamic getItemImage(String url) {
+    if (url != null) {
+      return url == "assets/images/canned-food.png"
+          ? AssetImage(url)
+          : NetworkImage(url);
+    } else {
+      return AssetImage("assets/images/canned-food.png");
+    }
+  }
+
+  String getUnitText(int unit) {
+    switch (unit) {
+      case 1:
+        return getString(context, 'item_unit');
+        break;
+      case 2:
+        return getString(context, 'item_liters');
+        break;
+      case 3:
+        return getString(context, 'item_grams');
+        break;
+      case 4:
+        return getString(context, 'item_kilos');
+        break;
+      default:
+        return getString(context, 'item_unit');
+        break;
+    }
+  }
+
+  getProductBrand(String completeName) {
+    if (completeName.contains('-')) {
+      return completeName.substring(completeName.indexOf('-') + 1);
+    }
+
+    return "";
+  }
+
+  getProductName(String completeName) {
+    if (completeName.contains('-')) {
+      return completeName.substring(0, completeName.indexOf('-'));
+    }
+    return completeName;
+  }
+
+  emptyRecipe() {
+    return SingleChildScrollView(
+      child: Center(
+        child: Column(
```

Comment:
remove all Containers and MediaQuery sizes. Just let Image.asset, Icon, Text and add mainAxisAlignment.SpaceEvenly on Column